metricbeat/scripts/config_collector.py

```diff
@@ -61,6 +70,20 @@ def collect(short=False):
     # output string so it can be concatenated
     print config_yml
 
+# Makes sure every title line is 80 chars long
```

Comment:
Ok, that means also below I have to take chars from 0:78 to have 80 including the newline.